platform_versioned_docs/version-23.4/cli/commands.mdx

```diff
@@ -0,0 +1,639 @@
+---
+title: "Commands"
+description: "Seqera Platform CLI commands."
+date: "16 Jul 2024"
+tags: [cli, commands]
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+:::note
+The CLI performs operations in the user workspace context by default. Use the `TOWER_WORKSPACE_ID` environment variable or the `--workspace` parameter to specify an organization workspace ID.
+:::
+
+Use the `-h` or `--help` parameter to list the available commands and their associated options.
+
+![`tw --help`](./_images/tw-info.svg)
+
+For help with a specific subcommand, run the command with `-h` or `--help` appended. For example, `tw credentials add google -h`.
+
+:::tip
+Use `tw --output=json <command>` to dump and store Seqera Platform entities in JSON format.
+
+Use `tw --output=json <command> | jq -r '.[].<key>'`  to pipe the command to use jq to retrieve specific values in the JSON output. For example, `tw --output=json workspaces list | jq -r '.workspaces[].orgId'` returns the organization ID for each workspace listed.
+:::
+
```

Comment:
The following `tw` commands don't seem to be documented at all:

- collaborators
- generate-completion
- info
- labels
- members
- teams